core/src/main/java/org/elasticsearch/plugins/RemovePluginCommand.java

```diff
@@ -56,8 +57,19 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
         execute(terminal, arg, env);
     }
 
-    // pkg private for testing
-    void execute(Terminal terminal, String pluginName, Environment env) throws Exception {
+/**
+    * Remove the plugin named pluginName.
```

Comment:
In fact, it should probably say something like `Remove the plugin specified by {@code pluginName}.`